app/src/main/java/com/github/se/orator/ui/overview/SpeakingPracticeModule.kt

```diff
@@ -91,29 +92,35 @@ fun SpeakingPracticeModule(
                     text = headerText,
                     style =
                         AppTypography.mediumTitleStyle.copy(
-                            fontWeight = FontWeight.Bold, color = AppColors.textColor),
+                            fontWeight = FontWeight.Bold,
+                            color = MaterialTheme.colorScheme.primary),
                     modifier =
                         Modifier.padding(vertical = dimensions.paddingMedium).testTag("titleText"))
 
                 // Dynamically generated input fields based on the provided data
                 inputs.forEach { input ->
-                  OutlinedTextField(
+                  androidx.compose.material.OutlinedTextField(
                       value = input.value,
                       onValueChange = input.onValueChange,
-                      label = { Text(input.label, color = AppColors.textColor) },
-                      placeholder = { Text(input.placeholder, color = AppColors.textColor) },
+                      label = { androidx.compose.material.Text(input.label) },
```

Comment:
maybe fix the import ?

Reply:
Also, are you using the default color by choice or by mistake ?

Reply:
Done !